### ISSUE TITLE
Please make releases

### ISSUE THREAD

**Issue by mattst88** [KhronosGroup/OpenCL-CLHPP#84]
It's been close to 4 years since the v2.0.10 tag was made. Linux distributions enjoy being able to ship tagged releases because it helps identify what software is included and makes cross-distro comparisons much simpler.

Nothing more needs to be done from your side than pushing a git tag.

**Comment by bashbaug** [comment]
@kpet, I think you were investigating this for the OpenCL headers:

https://github.com/KhronosGroup/OpenCL-Headers/issues/56

Can we use a similar convention here?  Is anything holding this up?

**Comment by kpet** [comment]
Yes. This project already had a versioning scheme in place and is integrated into multiple distributions so I'd rather not change that right now.

@mattst88  I've just created a release:https://github.com/KhronosGroup/OpenCL-CLHPP/releases/tag/v2.0.11. I wanted to sort out https://github.com/KhronosGroup/OpenCL-CLHPP/issues/78 before making more releases but I've got no reason to believe previous releases came with more than the testing we're currently doing. Worst case, we can make another one soon :).
